lib/jake/ref.ex

```diff
@@ -1,7 +1,60 @@
 defmodule Jake.Ref do
   alias Jake.Context
 
+  def expand_ref(%Context{child: %{"$ref" => ref} = spec} = context)
+      when ref == "#" do
+    new_child = Map.drop(spec, ["$ref"]) |> Map.merge(context.root)
+    {%{context | child: new_child}, true}
+  end
+
+  def expand_ref(%Context{child: %{"$ref" => ref} = spec} = context) when is_binary(ref) do
+    uri_parse = URI.decode(ref) |> URI.parse()
```

Comment:
you could just call it uri? uir_parse sounds like a verb

Reply:
maybe we should call it `parsed_uri`, in order to make distinction between raw uri and parsed uri with components.